### PR TITLE
Add default MFV config to standalone and parachain

### PR DESCRIPTION
**Pull request by petscheit**
fixes circuit panic issue in MFV

Locally, I still can't build without removing the `dev-dependency` flags in pallet-circuit. Anybody facing the same?

### CODE REVIEW

File: node/standalone/src/chain_spec.rs [hunk 1]
```diff
@@ -99,7 +100,7 @@ fn fetch_xdns_record_from_rpc(
                 genesis_hash: client.genesis_hash.0.to_vec(),
             },
             gateway_sys_props,
-            vec![],
+            vec![*b"tran"],
```
Comment (chiefbiiko) [inline node/standalone/src/chain_spec.rs:103]: is this change on purpose?

**Review by chiefbiiko — DISMISSED** [review]
🎉 🎉

File: node/standalone/src/chain_spec.rs (hunk 1 above)
Comment (petscheit) [reply node/standalone/src/chain_spec.rs:103]: ish, in preparation for the transfer demo. But I guess we have to register the gateway manually for now, so maybe it should go

Comment (chiefbiiko) [reply node/standalone/src/chain_spec.rs:103]: cool

File: runtime/parachain/src/lib.rs [hunk 2]
```diff
@@ -509,9 +509,12 @@ construct_runtime!(
         MultiFinalityVerifierGenericLike: pallet_mfv::<Instance4>::{
             Pallet, Call, Storage, Config<T, I>
         } = 104,
-        ContractsRegistry: pallet_contracts_registry::{Pallet, Call, Config<T>, Storage, Event<T>} = 105,
-        CircuitPortal: pallet_circuit_portal::{Pallet, Call, Storage, Event<T>} = 106,
-        Circuit: pallet_circuit::{Pallet, Call, Storage, Event<T>} = 107,
+         MultiFinalityVerifierDefault: pallet_mfv::{
```
Comment (MaciejBaj) [inline runtime/parachain/src/lib.rs:512]: extra space here?

**Review by MaciejBaj — APPROVED** [review]

File: runtime/parachain/src/lib.rs (hunk 2 above)
Comment (petscheit) [reply runtime/parachain/src/lib.rs]: cargo fmt was complaining if it left it 🤷‍♂️
